service/evidence/evidence_store_test.go

```diff
@@ -706,3 +901,22 @@ func TestService_GetEvidence(t *testing.T) {
 		})
 	}
 }
+
+func newAuthorizationStrategy(cloudServiceIDs []string) service.AuthorizationStrategy {
```

Comment:
Just a minor thing. This could actually be interesting for multiple tests so it might make sense to move this into into the `testutil` package. Otherwise I think we can finally merge this for now.

Reply:
@lebogg

Reply:
Yes, I did it initially but circular dependencies ...

Reply:
Maybe you can outsource the `AuthorizationStrategy` interface to API or internal?